tests/examples/test-app-advanced-extensions/pom.xml

```diff
@@ -0,0 +1,103 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+    JBoss, Home of Professional Open Source
+    Copyright 2015, Red Hat, Inc. and/or its affiliates, and individual
+    contributors by the @authors tag. See the copyright.txt in the
+    distribution for a full listing of individual contributors.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>openshift-demo</groupId>
+    <artifactId>demo-postgres</artifactId>
+    <packaging>war</packaging>
+    <version>1.0</version>
+    <name>OpenshiftDemo</name>
+    
+    <properties>
+        <maven.compiler.source>1.8</maven.compiler.source>
+        <maven.compiler.target>1.8</maven.compiler.target>
+    </properties>
+  
+    <description>This project demonstrates how to implement a JAX-RS service that uses JPA persistence to manage tasks</description>
+
+    <licenses>
+        <license>
+            <name>Apache License, Version 2.0</name>
+            <url>http://www.apache.org/licenses/LICENSE-2.0.html</url>
+            <distribution>repo</distribution>
+        </license>
+    </licenses>
+
+    <dependencies>
+        <dependency>
+            <groupId>javax.enterprise</groupId>
```

Comment:
Just wondering if we could have problems with the tests using these libraries. They are binary compatible with the ones shipped into CD19, although the versions are quite old. If those tests are not meant to be backported, looks reasonable to me to use the most accurate ones according to the current release, hence, the Jakarta ones. On the other hand, since they are binary compatible we could keep them, to facilitate the backporting.

Do you have any feedback regarding this?

Reply:
@yersan , these dependencies are provided, if the application compiles then we are fine. I would stick with older releases, but we could revisit if this happens to be an issue.